src/css/custom.css

```
@@ -1,30 +1,225 @@
+  color: var(--ifm-color-primary);
+}
+
+.theme-doc-version-badge {
+  background-color: rgba(242, 140, 56, 0.15);
+  color: var(--ifm-color-primary);
+}
+
+.alert--info {
+  --ifm-alert-background-color: rgba(242, 140, 56, 0.1);
+  --ifm-alert-color: var(--ifm-color-primary);
+  border-left-color: var(--ifm-color-primary);
```

Stylelint (17.2.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)